manifests/deploy/openshift/olm/bundle/4.7/meteringoperator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -207,15 +207,14 @@ metadata:
     categories: OpenShift Optional, Monitoring
     certified: "false"
     containerImage: quay.io/openshift/origin-metering-ansible-operator:4.7
-    createdAt: 2019-01-01T11:59:59Z
+    createdAt: "2019-01-01T11:59:59Z"
```

Comment:
I recognize this change as something a newer Helm would allow (force?). Is the removal of trailing new-lines also something newer Helm forced?

Reply:
I think that was just the expected behavior of `toYAML` in the helm 2.x world: https://github.com/helm/helm/issues/3470.

That said, not entirely sure if there's any impact to double-quoting the `createdAt` timestamp. I haven't seen anything pop up in the e2e logs that would suggest that OLM cannot consume that field in that format.